src/main/java/org/apache/sling/caconfig/ConfigurationBuilder.java

```diff
@@ -86,4 +86,11 @@
      */
     @NotNull <T> Collection<T> asAdaptableCollection(@NotNull Class<T> clazz);
 
+    /**
+     * Check if the configuration node is existing
+     * @param clazz Class that can be adapted from a {@link org.apache.sling.api.resource.Resource}
+     * @param <T> Annotation class type
+     * @return True/False based on configuration resource node existence in JCR.
+     */
+    @NotNull <T> boolean has(@NotNull Class<T> clazz);
```

Comment:
You need to extend the javadoc for „name“ as well, as setting the name is optional for this method as well.

Reply:
done

Reply:
I actually meant the javadoc in line 35ff

Reply:
Sorry, I adjusted it properly now.